### PR TITLE
feat: toggle hotkey notifications

### LINKED ISSUE #2245: Add option to show/hide notifications upon usage of hotkeys
As a user I want to toggle the notifications for the usage of hotkeys in the "Appearance" tab of the settings dialog. By default it should be turned on.

### PR DESCRIPTION
**Pull request by brandstetterm**
### Description
As a user I want to toggle the notifications for the usage of hotkeys in the "Appearance" tab of the settings dialog. By default it should be turned on.

Closes #2245 

### Changelog
- Add notifications toggle to the "Appearance" tab of the settings dialog
- Persist notifications enabled state in localStorage
- Add `dispatchNotification` function to HotkeyAnchor component that automatically dispatches a toast if notifications are enabled
- Improve accessibility of settings buttons / toggles 

### Visual Changes
localhost_3000_board_526d2377-5239-4af9-854b-7f5c584a1039_settings_appearance(iPad Mini)

localhost_3000_board_526d2377-5239-4af9-854b-7f5c584a1039_settings_appearance(iPad Mini) (1)

### CODE REVIEW

File: src/components/HotkeyAnchor/HotkeyAnchor.tsx [hunk 1]
```diff
@@ -61,13 +62,19 @@ export const HotkeyAnchor = () => {
     enabled: state.hotkeysAreActive && isAdmin,
   };
 
+  function dispatchHotkeyNotification(title: string) {
+    if (state.hotkeyNotificationsEnabled) {
+      Toast.info({title, autoClose: TOAST_TIMER_SHORT});
+    }
+  }
```
Comment (Schwehn42) [inline src/components/HotkeyAnchor/HotkeyAnchor.tsx:69]: This should be an arrow function, like the others
```suggestion
  const dispatchHotkeyNotification = (title: string) => {
    if (state.hotkeyNotificationsEnabled) {
      Toast.info({title, autoClose: TOAST_TIMER_SHORT});
    }
  };
```

File: src/store/middleware/view.tsx [hunk 2]
```diff
@@ -23,4 +23,16 @@ export const passViewMiddleware = (stateAPI: MiddlewareAPI<Dispatch, Application
         });
       });
   }
+
+  if (action.type === Action.EnableHotkeyNotifications) {
+    if (typeof window !== undefined) {
+      window.localStorage.setItem("hotkeyNotificationsEnabled", JSON.stringify(true));
```
Comment (Schwehn42) [inline src/store/middleware/view.tsx:29]: all locale storage keys have `"scrumlr/"` as prefix (except `theme`), maybe you should add it for consistency

File: src/store/middleware/view.tsx [hunk 3]
```diff
@@ -23,4 +23,16 @@ export const passViewMiddleware = (stateAPI: MiddlewareAPI<Dispatch, Application
         });
       });
   }
+
+  if (action.type === Action.EnableHotkeyNotifications) {
+    if (typeof window !== undefined) {
+      window.localStorage.setItem("scrumlr/hotkeyNotificationsEnabled", JSON.stringify(true));
```
Comment (Schwehn42) [inline src/store/middleware/view.tsx:29]: Please use `constants/storage.ts` and `utils/storage.ts` to access the locale storage

Comment (brandstetterm) [reply src/store/middleware/view.tsx]: Done ✅

File: src/store/reducer/view.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ const INITIAL_VIEW_STATE: ViewState = {
   feedbackEnabled: false,
   hotkeysAreActive: true,
   noteFocused: false,
+  hotkeyNotificationsEnabled: typeof window !== "undefined" && window.localStorage.getItem("scrumlr/hotkeyNotificationsEnabled") !== "false",
```
Comment (Schwehn42) [inline src/store/reducer/view.ts:11]: Also, here we should also use the `getFromStorage` function provided by `utils/storage.ts`

**Review by Schwehn42 — APPROVED** [review]
nice, looking good 👍

**Pull request merged**